learning_resources/views_userlist_test.py

```diff
@@ -283,3 +283,25 @@ def test_get_resource_user_lists(client, user, is_author, is_unlisted):
             assert item.get("child") == course.learning_resource.id
     else:
         assert items_json == []
+
+
+def test_set_userlist_relationships(client, user):
+    """Test the userlists endpoint for setting multiple userlist relationships"""
```

Comment:
**Request:** I think it's definitely worth testing "User X changes relationship does not affect other users' relationships" (earlier behavior in this PR.)

**Suggestion:** To this test, add something like
```python 
userlist_other_author = factories.UserListFactory.create()
assert userlist_other_author.author != userlists[0].author # I like this for sanity 🤷
factories.UserListRelationshipFactory.create(
    parent=userlist_other_author, child=course.learning_resource
)
# Then at the end, assert child still belongs to userlist_other_author
```